src/ggsn_gn.erl

```diff
@@ -279,17 +279,18 @@ handle_pdu(ReqKey, #gtp{ie = PDU} = Msg, _State,
     ergw_gsn_lib:ip_pdu(PDU, Context, PCtx),
     {keep_state, Data}.
 
+%% User Plane Error Indication
 handle_sx_report(#pfcp{type = session_report_request,
 		       ie = #{report_type := #report_type{erir = 1}}},
 	    _State, Data) ->
-    close_pdp_context(normal, Data),
+    close_pdp_context(session_error, Data),
```

Comment:
wrong naming, A GTP Error Indication, mean the remote peer didn't know about the GTP-U tunnel anymore, could be a remote failure or a peer restart.